lib/src/sql/number.rs

```diff
@@ -1,6 +1,7 @@
 use crate::sql::ending::number as ending;
 use crate::sql::error::IResult;
 use crate::sql::serde::is_internal_serialization;
+use bigdecimal::num_traits::Pow;
```

Comment:
Is this implemented?

Reply:
This import exposes the `pow` functionality for float, unfortunately not for `BigDecimal`